app/code/Magento/Customer/Model/CustomerRegistry.php

```diff
@@ -48,21 +54,30 @@ class CustomerRegistry
      */
     private $storeManager;
 
+    /**
+     * @var CustomerResourceFactory
+     */
+    private $customerResourceFactory;
+
     /**
      * Constructor
      *
      * @param CustomerFactory $customerFactory
      * @param CustomerSecureFactory $customerSecureFactory
      * @param StoreManagerInterface $storeManager
+     * @param CustomerResourceFactory $customerResourceFactory
      */
     public function __construct(
         CustomerFactory $customerFactory,
         CustomerSecureFactory $customerSecureFactory,
-        StoreManagerInterface $storeManager
+        StoreManagerInterface $storeManager,
+        CustomerResourceFactory $customerResourceFactory = null
```

Comment:
1. This class isn't marked as `@api`, so as part of 2.4.0 we should add NOT optional arguments.
2. Resource model shouldn't contain any data inside, so there is no need to use factory for this model. 
3. it's better name variable as `$customerResourceModel` to make it more clear

Please adjust it like this:
```suggestion
        CustomerResource $customerResourceModel = null
```